Package.swift

```diff
@@ -0,0 +1,17 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Solar",
+    platforms: [.iOS(.v8), .macOS(.v10_15), .watchOS(.v4), .tvOS(.v11)],
```

Comment:
Our .podspec supports macos 10.9, watchOS 3 and tvOS 9. According to [this page](https://docs.swift.org/package-manager/PackageDescription/PackageDescription.html), macOS support starts at 10.10.

Are we able to change these to `.macOS(.v10_10), .watchOS(.v3), .tvOS(.v9)`? My SPM knowledge is weak, so forgive me if these are not possible.

Reply:
Agreed, the minimum deployment targets should be as consistent as possible across dependency managers.

In https://github.com/ceeK/Solar/pull/47#discussion_r524836976, I had pushed back against setting iOS 12 as the minimum deployment target, so that part was dropped from the PR. But I think what you’re suggesting makes sense.